transcriptic/commands.py

```diff
@@ -1481,22 +1482,64 @@ def execute(
     api,
     workcell_id,
     device_set,
+    session_id,
     time_limit,
+    schedule_at,
     partition_group_size,
     partition_horizon,
     partitioning_swap_device_id,
 ):
+    # Define the initial payload
+    payload = {"timeLimit": f"{time_limit}:second"}
+
+    # Get the requested scheduling time
+    if schedule_at is not None:
+        if re.search(r"^\+\d+$", schedule_at):  # relative time
+            # take the next minutes!
+            parsed_schedule_at = (
+                int(time.time() // 60 + 1) + int(schedule_at[1:])
+            ) * 60
+        else:  # absolute time
+            current_time = time.localtime()
+            if re.search(
+                r"^\d{1,2}:\d{1,2}$", schedule_at
+            ):  # only time is given use today's year, month, and day
+                schedule_at_fixed = f"{current_time.tm_year}-{current_time.tm_mon}-{current_time.tm_mday}T{schedule_at}"
+            elif re.search(
+                r"^\d{2,2}T\d{1,2}:\d{1,2}$", schedule_at
+            ):  # use today's year, and month
+                schedule_at_fixed = (
+                    f"{current_time.tm_year}-{current_time.tm_mon}-{schedule_at}"
+                )
+            elif re.search(
+                r"^\d{1,2}-\d{1,2}T\d{1,2}:\d{1,2}$", schedule_at
+            ):  # use today's year
+                schedule_at_fixed = f"{current_time.tm_year}-{schedule_at}"
+            else:
+                schedule_at_fixed = schedule_at
+
+            try:
+                expected_schedule_at = datetime.strptime(
+                    schedule_at_fixed, "%Y-%m-%dT%H:%M"
+                )
+                parsed_schedule_at = int(expected_schedule_at.timestamp())
+            except ValueError:
+                click.echo(
+                    f"Couln't parse absolute time input, got {schedule_at} (fixed to {schedule_at_fixed})",
```

Comment:
nit: Couldn't

Reply:
Let's also restate the expected format in the error message so that its clearer what went wrong